Tasks/PackerBuildV1/src/operations/packerBuild.ts

```diff
@@ -63,7 +63,7 @@ function setOutputVariables(packerHost: packerHost, outputs: Map<string, string>
                     throw tl.loc("ImageURIOutputVariableNotFound");
                 }
             } else {
-                imageUri = outputs.get(constants.PackerLogTokenManagedImageName);
+                imageUri = outputs.get(constants.PackerLogTokenManagedImageLocation);
```

Comment:
This should be imageID, image location will lead to storing location such as " East US" in the imageUri output variable
Also we shall add a new output variable called imageId so that back compatibility is maintained.